src/python/pants/backend/codegen/thrift/python/py_thrift_namespace_clash_check.py

```diff
@@ -33,6 +34,9 @@ def register_options(cls, register):
     # TODO: deprecate the --strict option in a future release!
     register('--strict', type=bool, default=False, fingerprint=True,
              help='Whether to fail the build if thrift sources have invalid python namespaces.')
+    register('--dump-invalid-thrift-source-errors-inline', type=bool, default=True,
+             help='Whether to print all errors detecting namespaces in thrift sources to the '
```

Comment:
(Starting convo here for a distinct thread)

I think this option would be better named `--dump-errors-inline` or `--dump-errors-to-console`. It is already namespaced to Thrift, so saying `thrift-source-errors` seems repetitive to me.

If you want to clarify it's not all errors and only ones related to `--strict=True`, maybe `--dump-validation-errors-to-console`.

Reply:
`--dump-validation-errors-to-console` sounds perfect!